homework/js-exercises/Cat Walk/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title>Cat Walk</title>
+</head>
+
+<body>
+    <img style="position:absolute;" src="http://www.anniemation.com/clip_art/images/cat-walk.gif" />
+
+    <script>
+        let image = document.querySelector('img');
+        image.setAttribute('style', 'margin-left: 0px;');
+
+        const catWalk = () => {
+            positionOfCat = window.getComputedStyle(image).marginLeft.substring(0, window.getComputedStyle(image).marginLeft.length - 2);
+            positionInNum = parseInt(positionOfCat);
```

Comment:
You should always use a variable keyword (var, const, let) when defining a variable.

Reply:
For some reason (probably because it is in the HTML file?) I had no errors, usually the error appears if it is in a JS file and so I correct it.

Reply:
The best thing to do now that you are starting with js is always to place your code in a js file and enable the strict mode. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode